doc/flux_to_redux.md

```diff
@@ -37,7 +37,7 @@ No. OverReact Redux is meant to help to provide a recommended state management m
 To evaluate if the refactor is worth it, the details of OverReact Redux can be found in the [OverReact Redux documentation](./over_react_redux_documentation.md). That document paired with this guide will illustrate the full scope of costs and benefits.
 
 ## A Simple Example
-To illustrate the basic changes that will occur, this section will go through a basic Flux architecture and then show that same system with Redux instead. This section will also have a step by step list of instructions for an update as straightforward as this one. An actual working example can be found in the [Flux to Redux Example](../web/flux_to_redux/simple/readme.md).
+To illustrate the basic changes that will occur, this section will go through a basic Flux architecture and then show that same system with Redux instead. This section will also have a step by step list of instructions for an update as straightforward as this one. An actual working example can be found in the [Flux to Redux Example](../web/flux_to_redux/simple).
```

Comment:
This change was actually a topic of conversation:
https://github.com/Workiva/over_react/pull/438#discussion_r369290885

Reply:
I'm happy to go either way, just FYI! :)